test/tests/builtin_type_not_subclassable.py

```diff
@@ -1,3 +1,19 @@
 # should_error
+import re
+
 class foo(slice):
    pass
+
+class C(type(xrange(1))):
```

Comment:
I don't think any of this code will get run, since we will just exit at the first uncaught exception.